src/main/java/recipeditor/parser/FlagParser.java

```diff
@@ -29,6 +32,10 @@ public class FlagParser {
     private static final String TITLE_FLAG = FLAG_INITIAL + "t";
     private static final String DESCRIPTION_FLAG = FLAG_INITIAL + "d";
     private static final String INDEX_FLAG = FLAG_INITIAL + "id";
+    private static final String COMMAND_WORD_EDIT = "/edit";
```

Comment:
Maybe instead of creating another variable for "/edit", you can take this from EditCommand.COMMAND_TYPE.